eventmesh-registry-plugin/eventmesh-registry-etcd/src/main/java/org/apache/eventmesh/registry/etcd/service/EtcdRegistryService.java

```diff
@@ -204,7 +206,7 @@ public boolean register(EventMeshRegisterInfo eventMeshRegisterInfo) throws Regi
             EventMeshDataInfo eventMeshDataInfo =
                 new EventMeshDataInfo(eventMeshClusterName, eventMeshName,
                     endPoint, System.currentTimeMillis(), eventMeshRegisterInfo.getMetadata());
-            ByteSequence etcdValue = ByteSequence.from(JsonUtils.toJSONString(eventMeshDataInfo).getBytes(Constants.DEFAULT_CHARSET));
+            ByteSequence etcdValue = ByteSequence.from(Objects.requireNonNull(JsonUtils.toJSONString(eventMeshDataInfo)).getBytes(Constants.DEFAULT_CHARSET));
```

Comment:
This line is too long and will cause the checkstyle check to fail, so suggest a line break.

Reply:
Thanks for your help.